dotnet/src/Skills/Skills.OpenAPI/Extensions/KernelChatGptPluginExtensions.cs

```diff
@@ -176,4 +206,12 @@ private static string ParseOpenApiUrl(string gptPluginJson)
 
         return openApiUrl!;
     }
+
+    private static PluginManifest ParsePluginJson(string pluginJson)
+    {
+        var manifest = JsonSerializer.Deserialize<PluginManifest>(pluginJson, new JsonSerializerOptions { IncludeFields = true })
```

Comment:
The PluginManifest class/model does not declare any fields, so setting the flag IncludeFields may not be necessary.